src/modules/email.py

```diff
@@ -0,0 +1,8 @@
+from src.modules.field import Field
+
+
+# TODO: Improvement. Refactor. Add error handler decorator
+
+class Email(Field):
+    def __init__(self, value):
+        super().__init__(value)
```

Comment:
rename value to email
add class attribute email.

    def __init__(self, email):
        super().__init__(email)
        self.email = Field(email)